tests/integration/base.py

```diff
@@ -124,22 +124,96 @@ def _create_session(cursor, session_id, user_id):
 
 def _create_global_topics(cursor):
     # type: (psycopg2.extensions.cursor) -> None
+
     cursor.execute(
-        "insert into"
-        " phpbb_topics(topic_id, forum_id, topic_title, topic_time,"
-        "  topic_first_poster_name, topic_first_post_id, topic_type)"
-        " values (0,0,'naslov_teme_0',10,'ime',0,3),"
-        " (1,0,'naslov teme 1',13,'drugi poster',1,3),"
-        " (2,0,'naslov teme 2',200,'post it',2,3),"
-        " (3,0,'naslov teme 3',256,'posted it',3,3)"
+        "insert into phpbb_topics ("
+        "topic_id"
+        ",forum_id"
+        ",topic_title"
+        ",topic_time"
+        ",topic_first_poster_name"
+        ",topic_first_post_id"
+        ",topic_type"
+        ") values ("
+        "0"
+        ",0"
+        ",'topic title 0'"
+        ",10"
+        ",'name'"
+        ",0,3"
+        "), ("
+        "1,"
+        "0,"
+        "'topic title 1',"
+        "13,"
+        "'second poster',"
+        "1,"
+        "3"
+        "), ("
+        "2"
+        ",0"
+        ",'topic title 2'"
+        ",200"
+        ",'post it'"
+        ",2,"
+        "3"
+        "), ("
+        "3"
+        ",0"
+        ",'topic title 3'"
+        ",256"
+        ",'posted it'"
+        ",3"
+        ",3"
+        "), ("
+        "4"
+        ",0"
+        ",'topic title 4'"
+        ",666"
+        ",'posted it again'"
+        ",3"
+        ",3"
+        ") , ("
+        "6"
+        ",0"
+        ",'topic title 5'"
+        ",777"
+        ",'posted as 7, expecting 6'"
+        ",3"
+        ",3"
+        ") , ("
+        "7"
+        ",0"
+        ",'different data type'"
+        ",999"
+        ",'inserted into table, expecting different type'"
+        ",3"
+        ",3"
```

Comment:
Should this number be something else (topic_type)?

I see in your dataset you have only topic_type 3.

Reply:
I have set up one example where topic_type is different from 3. But the select expression for global topics has a where clause with a condition that topic_type = 3. So I think it would make sense to have most of examples fulfilling this condition.